File: backend/NEOBTE/src/main/java/com/sesame/neobte/DTO/RegisterRequest.java
package com.sesame.neobte.DTO;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
public class RegisterRequest {
    private String email;
    private String nom;
    private String prenom;
    private String adresse;
    private Integer age;
    private String job;
    private String genre;
    private String motDePasse;
}

File: frontend/NEOBTE/src/app/Entities/Interfaces/register-request.ts
export interface RegisterRequest {
  email: string;
  username: string;
  motDePasse: string;
  nom: string;
  prenom: string;
  cin: string;
  telephone: string;
  dateNaissance: string;
  job: string;
  genre?: 'HOMME' | 'FEMME';
  adresse?: string;
  codePostal?: string;
  pays?: string;
}
